src/test/kotlin/com/expedia/graphql/execution/FunctionDataFetcherTest.kt

```diff
@@ -31,6 +34,13 @@ internal class FunctionDataFetcherTest {
             delay(10)
             string
         }
+
+        fun throwException() { throw GraphQLException("Test Exception") }
+
+        suspend fun suspendThrow(): CompletableFuture<*> = coroutineScope {
```

Comment:
when using coroutines the whole point would be to avoid `CompletableFuture`, lets just change it to `String`